src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/cluster-createdDate-default.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="kumra" id="cluster-createdDate-default">
+        <sql dbms="mysql">ALTER TABLE  CLUSTER
+            CHANGE createdDate createdDate TIMESTAMP(6) NOT NULL DEFAULT 0</sql>
```

Comment:
I'm curious why this is needed. Looks like the current default is `CURRENT_TIMESTAMP(6)`

Reply:
MySQL implicitly adds a constraint that sets the timestamp field to the current timestamp on update, this is to get around it.

https://stackoverflow.com/questions/22860351/mysql-column-type-timestamp-implicitly-includes-not-null-default-current-time

Reply:
Wow, wtf MySQL. Thanks :)

Reply:
Yeesh.  Good catch.